src/App.jsx

```diff
@@ -14,45 +15,68 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
-
-    <ul>
-      <li data-cy="Good">Dumplings</li>
-      <li data-cy="Good">Carrot</li>
-      <li data-cy="Good">Eggs</li>
-      <li data-cy="Good">Ice cream</li>
-      <li data-cy="Good">Apple</li>
-      <li data-cy="Good">...</li>
-    </ul>
-  </div>
-);
+export const App = () => {
+  const [sortedGoods, setSortedGoods] = useState(goodsFromServer);
+
+  const sortAlphabetically = () => {
+    const sorted = [...sortedGoods].sort();
+
+    setSortedGoods(sorted);
+  };
+
+  const sortLength = () => {
+    const sorted = [...sortedGoods].sort((a, b) => a.length - b.length);
+
+    setSortedGoods(sorted);
+  };
+
+  const sortReverese = () => {
+    const sorted = [...sortedGoods].reverse();
+
+    setSortedGoods(sorted);
+  };
+
+  return (
+    <>
```

Comment:
The fragment should be used only if the rendered elements are not wrapped in one parent element.